src/main/kotlin/com/hibob/feedback/resource/FeedbackResource.kt

```diff
@@ -3,31 +3,52 @@ package com.hibob.feedback.resource
 import com.hibob.feedback.service.FeedbackService
 import com.hibob.feedback.dao.*
 import jakarta.ws.rs.*
+import jakarta.ws.rs.container.ContainerRequestContext
+import jakarta.ws.rs.core.Context
 import jakarta.ws.rs.core.Response
 import org.springframework.stereotype.Component
 import java.util.*
 import jakarta.ws.rs.core.MediaType
 
 
 @Component
-@Path("/feedback")
+@Path("/api/feedback")
 @Produces(MediaType.APPLICATION_JSON)
 @Consumes(MediaType.APPLICATION_JSON)
 class FeedbackResource(private val feedbackService: FeedbackService) {
 
     @POST
     @Path("/v1/create")
-    fun createFeedback(feedbackInput: FeedbackInput): Response {
-        val activeUser = ActiveUser(UUID.randomUUID(), UUID.randomUUID()) //TO DO: change it to take proprties from the header
+    fun createFeedback(feedbackInput: FeedbackInput, @Context requestContext: ContainerRequestContext): Response {
+        val activeUser = getActiveUserOrThrow(requestContext)
         val feedbackId = feedbackService.createFeedback(feedbackInput, activeUser)
         return Response.ok(feedbackId).build()
     }
 
+    private fun getActiveUserOrThrow(requestContext: ContainerRequestContext): ActiveUser {
+        return requestContext.getProperty("activeUser") as? ActiveUser
+            ?: throw BadRequestException("user is not an active user")
+    }
+
     @GET
     @Path("/v1/feedbackId/{feedbackId}")
-    fun getFeedback(@PathParam("feedbackId") feedbackId: UUID): Response {
-        val activeUser = ActiveUser(UUID.randomUUID(), UUID.randomUUID()) //TO DO: change it to take proprties from the header
+    fun getFeedback(
+        @PathParam("feedbackId") feedbackId: UUID,
+        @Context requestContext: ContainerRequestContext
+    ): Response {
+        val activeUser = requestContext.getProperty("activeUser") as? ActiveUser
+        activeUser ?: throw BadRequestException("user is not an active user")
```

Comment:
use the getActiveUserOrThrow method here too

Reply:
also make the "activeUser" string a const, either here or preferably in the AuthenticationFilter